src/python/ensembl/io/genomio/seq_region/prepare.py

```diff
@@ -498,6 +503,19 @@ def main() -> None:
     parser.add_argument(
         "--to_exclude", nargs="*", metavar="SEQ_REGION_NAME", help="Sequence region names to exclude"
     )
+    parser.add_argument("-d", "--debug", action="store_true", help="Debug level logging")
+    parser.add_argument("-v", "--verbose", action="store_true", help="Verbose level logging")
+
     args = parser.parse_args()
+    
+    log_format = "%(asctime)s\t%(levelname)s\t%(message)s"
+    date_fmt = r"%Y-%m-%d %H:%M:%S"
+    reload(logging)
+    if args.verbose:
+        logging.basicConfig(format=log_format, datefmt=date_fmt, level=logging.INFO)
+    elif args.debug:
+        logging.basicConfig(format=log_format, datefmt=date_fmt, level=logging.DEBUG)
+    else:
+        logging.basicConfig(format=log_format, datefmt=date_fmt, level=logging.WARNING)
 
-    prepare_seq_region_metadata(**vars(args))
+    prepare_seq_region_metadata(**vars(args))
```

Comment:
I think it's better to give the explicit list and handle the verbose/debug in the main, rather than pass them to prepare_seq_region_metadata()